src/App.js

```diff
@@ -1,25 +1,29 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+/* eslint-disable no-unused-vars */
```

Comment:
What is this for?

Reply:
eslint highlights variables that do not appear to be used and thus affect codacy. For example, React import. Eslint provides those inline comments to fix the issues.